config/Config-images.in

```diff
@@ -194,19 +194,28 @@ menu "Target Images"
 		select PACKAGE_grub2
 		default y
 
+	config EFI_IMAGES
+		bool "Build EFI images (Linux x86 or x86_64 host only)"
+		depends on TARGET_x86
+		depends on TARGET_ROOTFS_EXT4FS || TARGET_ROOTFS_ISO || TARGET_ROOTFS_JFFS2 || TARGET_ROOTFS_SQUASHFS
+		select PACKAGE_grub2
+		select PACKAGE_grub2-efi
```

Comment:
Where is `PACKAGE_grub2-efi` defined?

Reply:
PACKAGE_grub2-efi is defined in package/boot/grub2/grub2-efi/Makefile

Reply:
I don't have that.